obelix/idefix/devAuth/github.go

```diff
@@ -0,0 +1,43 @@
+package devAuth
+
+import (
+	"context"
+	"fmt"
+	"golang.org/x/oauth2"
+	"log"
+	"net/http"
+)
+
+func GithubAuth() *http.Client {
+	ctx := context.Background()
+	conf := &oauth2.Config{
+		ClientID:     "3569cf3d4c2b88ddc23e",
+		ClientSecret: "6d47da5ddb50575977f2f7b32381b217ae8ff785",
+		Scopes:       []string{"user", "repo"},
```

Comment:
You should take scopes as a parameter.
Consider using the same format as in panoramix, with a JSON configuration file read at the program start.

Reply:
ecf1b7497f231446533518632673432ae8e318ed